time stamp
geometry_msgs/Point[] unid_markers
geometry_msgs/Pose[] rigid_bodies
std_msgs/Int64[] rigid_body_ids
